assets/programs/vix_demo/Makefile

```
@@ -0,0 +1,48 @@
+FILES = ./build/start.asm.o ./build/vix_demo.o ./build/vios.asm.o ./build/vios.o ./build/stdlib.o ./build/stdio.o ./build/string.o ./build/memory.o ./build/start.o
+INCLUDES = -I./src -I../stdlib/src
+FLAGS = -g -falign-jumps -falign-functions -falign-labels -falign-loops -fstrength-reduce -fomit-frame-pointer -finline-functions -Wno-unused-function -fno-builtin -Werror -Wno-unused-label -Wno-cpp -Wno-unused-parameter -nostdlib -nostartfiles -nodefaultlibs -Wall -O0 -Iinc -std=gnu99
+STDLIB = ../stdlib
+
+all: vix_demo.elf
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)